drivers/modem/quectel-bg9x.c

```diff
@@ -1188,6 +1196,29 @@ static int modem_init(const struct device *dev)
 			(k_thread_entry_t) modem_rx,
 			NULL, NULL, NULL, K_PRIO_COOP(7), 0, K_NO_WAIT);
 
+
+        /* power on modem */
+        // Enable the battery voltage pin (VLTE-ENB)
+        uint8_t pinPosition = 1 + (8 * 1); // VLTE-ENB is pin 9 on the GPIO expander.
```

Comment:
The pin should be a variable (or defined using Device tree), because it can change on any board layout.